DbMocker.Tests/ConversionTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Apps72.Dev.Data.DbMocker;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+
+namespace DbMocker.Tests
+{
+    [TestClass]
+    public class ConversionTests
+    {
+        [TestMethod]
+        public void Mock_EmptyTable_Test()
+        {
+            var mockTable = MockTableExtensions.GetEmptyMockTable<RowDto>();
```

Comment:
I prefer to create a new typed MockTable using:
```csharp
public class MockTable<T> : MockTable
{
    ...
}
```

So, you will create a new table like that: `var mockTable = new MockTable<RowDto>();`
And you can also use a second possibility using `var mockTable = MockTable.Empty<RowDto>();`

Reply:
Then logics moves to the class' constructor.
I can do it and keep fluent methods.

Reply:
A lot of logic move to the class' constructor. It isn't ok. 
But  I can make it when we agree on other questions.